crates/bindings-csharp/Codegen/Module.cs

```diff
@@ -247,7 +247,7 @@ class {r.Name}: IReducer {{
                                     );
                                 }}
 
-                                void IReducer.Invoke(BinaryReader reader, SpacetimeDB.Runtime.DbEventArgs dbEvent) {{
+                                void IReducer.Invoke(BinaryReader reader, SpacetimeDB.Runtime.ReducerContext dbEvent) {{
                                     {r.FullName}({string.Join(", ", r.Args.Select(a => a.IsDbEvent ? "dbEvent" : $"{a.Name}.Read(reader)"))});
```

Comment:
This still references `"dbEvent"`, is that fine?

Reply:
Good catch. I stupidly replaced all instances of the class but not variable names.